<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-B3UTCUD" failures="2" tests="2" name="coverFoxTestPackage.TestExecutionUsingTestNg1" time="1.725" errors="0" timestamp="2024-08-10T14:46:13 IST" skipped="0">
  <testcase classname="coverFoxTestPackage.TestExecutionUsingTestNg1" name="validateSortFilter" time="1.664">
    <failure message=" sort filter is not displayed , tc fails expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError:  sort filter is not displayed , tc fails expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at coverFoxTestPackage.TestExecutionUsingTestNg1.validateSortFilter(TestExecutionUsingTestNg1.java:84)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
    <system-out>
      <![CDATA[validateSortFilter this test case is started
validating sort button
test case validateSortFilter is failed
taking screenshot
saving file at locationC:\Users\ADMIN\eclipse-workspace\.metadata\CoverFoxTest\screenshots\validateSortFilter20240810144601.jpg]]>
    </system-out>
  </testcase> <!-- validateSortFilter -->
  <testcase classname="coverFoxTestPackage.TestExecutionUsingTestNg1" name="validateBanners" time="0.061">
    <failure message="null" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.fail(Assert.java:116)
at coverFoxTestPackage.TestExecutionUsingTestNg1.validateBanners(TestExecutionUsingTestNg1.java:72)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
    <system-out>
      <![CDATA[validateBanners this test case is started
test case validateBanners is failed
taking screenshot
saving file at locationC:\Users\ADMIN\eclipse-workspace\.metadata\CoverFoxTest\screenshots\validateBanners20240810144559.jpg]]>
    </system-out>
  </testcase> <!-- validateBanners -->
  <system-out>
    <![CDATA[launching browser
waiting.....
clicking on next button
clicking on next button
reading data from excel file
handling age dropdown
clicking on next button
reading data from excel file
entering pincode
reading data from excel file
entering mobile no
clicking on next button
validateBanners this test case is started
test case validateBanners is failed
taking screenshot
saving file at locationC:\Users\ADMIN\eclipse-workspace\.metadata\CoverFoxTest\screenshots\validateBanners20240810144559.jpg
validateSortFilter this test case is started
validating sort button
test case validateSortFilter is failed
taking screenshot
saving file at locationC:\Users\ADMIN\eclipse-workspace\.metadata\CoverFoxTest\screenshots\validateSortFilter20240810144601.jpg
closing browser]]>
  </system-out>
</testsuite> <!-- coverFoxTestPackage.TestExecutionUsingTestNg1 -->
